pages/common/npm.md

```diff
@@ -4,34 +4,34 @@
 > Manage Node.js projects and their module dependencies.
 > Homepage: <https://www.npmjs.com/>.
 
-- Download and install a module globally:
+- Interactively create a package.json file:
 
-`npm install -g {{module_name}}`
+`npm init`
 
-- Download all dependencies referenced in package.json:
+- Download all the packages listed as dependencies in package.json:
 
 `npm install`
 
-- Download a given dependency required for the application to run, and add it to the package.json:
+- Download a package and add it to the list of dependencies in package.json:
 
-`npm install {{module_name}}@{{version}} --save`
+`npm install {{module_name}}`
```

Comment:
Don't you need to add `--save` for this to actually add to `package.json`? I'm not sure that removing this is a good idea.

Did this behaviour change at some point - could you point me at some documentation for this?

Reply:
No. Saving is the default behavior starting in npm 5. Relevant section of official docs: https://docs.npmjs.com/cli/install.html

Reply:
I see. Thanks! That'll be a hard habit to break lol :P